charts/k6-operator/values.schema.json

```diff
@@ -311,19 +386,6 @@
       "title": "podLabels",
       "type": "object"
     },
-    "prometheus": {
```

Comment:
@EladAviczer, I guess the confusion came from the `prometheus` section in here, in schema. As in, yesterday I could create values referencing `prometheus` and it passed validation. Now section is removed from schema and it doesn't pass it :slightly_smiling_face: